src/main/java/com/elixr/poc/rest/response/ExceptionHandler.java

```diff
@@ -0,0 +1,41 @@
+package com.elixr.poc.rest.response;
+
+import com.elixr.poc.rest.request.UserRequest;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.FieldError;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/*
+Handling the exception and sending propper error message
+ */
+@ControllerAdvice
+public class ExceptionHandler {
+    private ErrorResponse errorResponse;
+
+
+    @org.springframework.web.bind.annotation.ExceptionHandler(MethodArgumentNotValidException.class)
```

Comment:
The class name and the annotation name are the same, that is why you have to provide the class name with the package. So rename this class and change it accordingly

Reply:
Changed the class name to